packages/components/src/components/themed-scrollbars/themed-scrollbars.tsx

```diff
@@ -38,7 +40,7 @@ const ThemedScrollbars = ({
 
     return (
         <div
-            data-testid='dt_themed_scrollbars'
+            data-testid={testId}
```

Comment:
we shouldn't pass the test ids like this, but I think for backward compatibility we have to do it this way. is it possible to remove all of the hard coded test-ids from our codebase in a separate card?

Reply:
I agree . will create a card for that. thanks